protocol/statetracker/emergency_tracker.go

```diff
@@ -0,0 +1,84 @@
+package statetracker
+
+import (
+	"sync"
+	"time"
+
+	"github.com/lavanet/lava/utils"
+	downtimev1 "github.com/lavanet/lava/x/downtime/v1"
+)
+
+type EmergencyTracker struct {
```

Comment:
let's separate last epoch tracking from full epochs tracking the provider needs, having them together separated only by wether we call updateEpoch is confusing